controls/bsi_sys_1_1_rhel9.yml

```
@@ -0,0 +1,671 @@
+# In BSI Basic Protection are multiple Requirements in one control.
+# i.e. there are multiple sentences, some including a RFC2119 keyword
+# Since we must increase granularity to create a precise control,
+# we number each sentence with a RFC2119 keyword as a section, grouping sentences, which are logically connected.
+# we number inline in brackets, so the lookup is easy
+# we reference these numbers in comments over each rule or group of rules
+policy: 'BSI-SYS-1-1-RHEL9'
+title: 'SYS.1.1 General Server (RHEL9)'
+id: bsi_sys_1_1_rhel9
+version: '1.0'
+source: https://www.bsi.bund.de/SharedDocs/Downloads/EN/BSI/Grundschutz/International/bsi_it_gs_comp_2022.pdf
+levels:
+  - id: basic
+  - id: standard
+    inherits_from:
+      - basic
+  - id: elevated
+    inherits_from:
+      - standard
+
+reference_type: bsi
+product: rhel9
+
+controls:
+  - id: SYS.1.1.A1
+    title: Appropriate Installation
+    levels:
+      - basic
+    description: >-
+      (1) Servers MUST be operated in locations that may only be accessed by authorised persons.
+      (2) Servers MUST therefore be set up and installed in data centres, computer rooms, or lockable server rooms (see the corresponding modules in the INF Infrastructure layer). (3) Servers MUST NOT be used as personal computers (4) IT systems used as workstations MUST NOT be used as servers.
+    notes: >-
+      This requirement must be implemented organizationally and cannot be checked technically
+    status: manual
+
+  - id: SYS.1.1.A2
+    title: User Authentication on Servers
+    levels:
+      - basic
+    description: >-
```

```suggestion
    description: |-
```

Please convert all multi line to `|-` vs `>-`